scripts/migrate-test-names.mjs

```diff
@@ -26,6 +26,7 @@ Then for the couple deeper packages
   yarn lerna exec --concurrency 1 --no-bail '../../../scripts/migrate-test-names.mjs'
 
 `;
+console.log(usage);
```

Comment:
Seems odd to log usage on the happy path.

Reply:
agreed. it was expedient to fix a lint error that `usage` isn't ever used.